tracer/src/Datadog.Trace/FrameworkDescription.NetCore.cs

```diff
@@ -93,12 +93,19 @@ private static string GetNetCoreOrNetFrameworkVersion()
             {
                 try
                 {
+#if NETFX
```

Comment:
I think the "canonical" property is `NETFRAMEWORK`
```suggestion
#if NETFRAMEWORK
```